src/requested-data/requested-data-reducer.ts

```diff
@@ -18,23 +18,36 @@ export function requestedData(
                 ...state,
                 electionType: action.electionType,
                 enableAutoSave: true,
+                dataLoaded: checkStateLoaded(state, action),
```

Comment:
Codacy found an issue: ['checkStateLoaded' was used before it was defined.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5611900)